### PR TITLE
latest coverlet

### PR DESCRIPTION
**Pull request by clairernovotny**

### CODE REVIEW

**Comment by clairernovotny** [comment]
@MarcoRossignoli @tmat

Getting errors here using the 1.3.x nightly of the collectors

**Comment by clairernovotny** [comment]
I wonder if there's a dependency/ordering issue when the `VSTest` target is used:

image

The `InitializeSourceControlInformation` target shows as grey in that call path.

File: Directory.Build.targets [hunk 1]
```diff
@@ -7,4 +7,13 @@ https://github.com/dotnet/sourcelink/issues/572 -->
   <ItemGroup>
     <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
   </ItemGroup>
+  
+  <Target Name="CoverletGetPathMap"
```
Comment (MarcoRossignoli) [inline Directory.Build.targets:11]: Are you missing?
```xml
  <ItemGroup>
    <SourceRoot Include="$(NuGetPackageRoot)" />
  </ItemGroup>
```

Comment (clairernovotny) [reply Directory.Build.targets:11]: Shouldn't need that. That workaround is only needed for packages that contain source files.

**Pull request merged**